docs/index.html

```diff
@@ -0,0 +1,252 @@
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px] py-3">
+          <a href="index.html">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#home">
+            <i class="fa-solid fa-x fa-xl"></i>
+          </a>
+        </div>
+
+        <nav>
+          <ul class="flex flex-col gap-5 text-3xl font-bold px-6">
+            <li>
+              <a href="index.html">Home</a>
+            </li>
+
+            <li>
+              <a href="about.html">About</a>
+            </li>
+
+            <li>
+              <a href="#program">Program</a>
+            </li>
+
+            <li>
+              <a href="#join">Join</a>
+            </li>
+
+            <li>
+              <a href="#sponsor">Sponsor</a>
+            </li>
+
+            <li>
+              <a href="#news">News</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <!-- Hero Section -->
+
+      <section class="bg-hero-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-stone-300 pb-[100px] px-4 py-3 md:px-[5%] lg:px-[15%] md:py-[100px]">
+
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px]">
+          <a href="index.html">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#menu">
+            <i class="fa-solid fa-bars fa-xl"></i>
+          </a>
+        </div>
+
+        <h2 class="text-3xl text-[#ec5242] mb-2 md:text-5xl">
+          "Hello, Sharing world!"
+        </h2>
+
+        <h1 class="text-6xl font-extrabold text-[#ec5242]/[0.8] mb-6 lg:w-[80%] md:text-8xl">
+          Garden and Flower Show 2023
+        </h1>
+
+        <p class="text border-4 border-zinc-500 p-4 mb-4 md:w-[70%] lg:w-[55%] md:mb-8">
+          A joyful celebration of amazing garden designs, gorgeous floral displays and exclusive shopping deals.
+        </p>
+
+        <h2 class="text-4xl font-extrabold mb-2 md:mb-4">
+          2023.05.15. ~ 05.20.
+        </h2>
+        <h3 class="text-xl">
+          @ Hungary, Tiszakürti Arborétum
+        </h3>
+      </section>
+
+      <!-- Main Program Section -->
+
+      <section class="bg-main-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-[#272a31] text-[#d3d3d3] px-4 py-8 flex flex-col justify-evenly gap-8 md:py-10 md:px-[5%] lg:px-[15%]">
+        <h3 class="text-3xl font-bold text-center mb-6">
+          Main Program
+
+          <div class="border-b-2 border-[#ec5242] w-[60px] pt-4 mx-auto"></div>
+        </h3>
+
+        <div class="flex flex-col gap-4 justify-evenly md:flex-row md:gap-2" id="program-card"></div>
+
+        <a href="#program" class="block md:hidden text-xl text-center font-bold border-2 border-emerald-800 bg-emerald-800 p-6 w-[90%] mx-[5%] mb-6 mt-6">
+          Visit Flower and Garden Show 2023
+        </a>
```

Comment:
- [ ] Please ensure that the background colour of the cards in the main program section matches the original design, as requested by the previous reviewer. :ok_hand: 😊

- [ OPTIONAL ]  I believe that changing the background image of the main section to match the original design will make it easier to fix the issue with the cards' background colour.

### your design
![Screenshot from 2023-04-22 00-18-19](https://user-images.githubusercontent.com/89228743/233751750-87b0f7e3-1384-49a6-9d34-89764a32cdeb.png)

### original design
![Screenshot from 2023-04-22 00-20-38](https://user-images.githubusercontent.com/89228743/233751770-c699d5c4-086c-49bb-82c4-469c768e8679.png)